Svg generator DX fixes (#540)
vaadin/docs: articles/ds/components/charts/java-api/svg-generator.asciidoc

</dependency>
----

> .NodeJS required
> NOTE: NodeJS must be installed for the generator to work.

Reviewer: ```suggestion
.Node.js required
NOTE: Node.js must be installed for the generator to work.
```

// ...
try (SVGGenerator generator = new SVGGenerator()) {
    String svg = generator.generate(configuration);
} catch (IOException | InterruptedException ex) {
    // handle exceptions accordingly
}
----

== Customizing SVG Generation

> Some attributes about the resulting SVG can be customized. 
> Customizable options include:

Reviewer: ```suggestion
You can customize some attributes of the resulting SVG.

The customizable options include the following:
```

options.setHeight(600);
try (SVGGenerator generator = new SVGGenerator()) {
    String svg = generator.generate(configuration, options);
} catch (IOException | InterruptedException ex) {
    // handle exceptions accordingly
}
----

=== Executing JavaScript Functions

> It is possible to execute functions from [classname]#Configuration# objects, for example, formatter functions.
> Executing functions must be explicitly enabled by setting the `executeFunctions` flag to `true`.

Reviewer: ```suggestion
Your can execute functions from [classname]#Configuration# objects, for example, formatter functions.
Executing functions must be explicitly enabled by setting the `executeFunctions` flag to `true`.
```

Author: I guess you meant `You can`.

options.setExecuteFunctions(true);
try (SVGGenerator generator = new SVGGenerator()) {
    String svg = generator.generate(configuration, options);
} catch (IOException | InterruptedException ex) {
    // handle exceptions accordingly
}
----

== Preview SVG (For Debugging)

> You can add the SVG to a component to see the resulting image.
> 
> .For debugging purposes only
> CAUTION: Use the <<./basic-use#,Chart>> component to render charts.
> This approach is only intended to help you debug your app.

Reviewer: ```suggestion
You can add the generated SVG to a component to see the resulting image.

.For debugging purposes only
CAUTION: Use the <<./basic-use#,Chart>> component to render charts.
This approach is only intended to help you debug your application.
```
